src/chainparams.cpp

```diff
@@ -214,10 +214,10 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_V20].nFalloffCoeff = 5;            // this corresponds to 10 periods
 
         consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].bit = 10;
-        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nStartTime = 19999999999;   // TODO: To be determined later
-        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
+        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nStartTime = 1704088800;   // January 1, 2024
```

Comment:
That's  `Mon Jan 01 2024 06:00:00 GMT+0000` and we usually use midnight

Reply:
do you want it changed? can you provide a new timestamp(s)?

Reply:
>>> 1704088800 - 60 * 60  * 6
1704067200
is it?

Reply:
```suggestion
        consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].nStartTime = 1704067200;   // January 1, 2024
```